src/Main.java

```diff
@@ -15,32 +15,33 @@ public static void main(String[] args) throws Exception {
     System.out.println("You have " + letsPlay.getGuessesLeft() + " guesses left");
     System.out.println(letsPlay.getRevealedMovieTitle());
 
-    while (letsPlay.isGameInPlay()) {
+    while (!letsPlay.isHasWon() && !letsPlay.isLastGuess()) {
 
       Scanner userInput = new Scanner(System.in);
 
       letsPlay.playturn(userInput);
```

Comment:
playTurn could return a boolean which signifies if the last guess was correct or not.
```
if (letsPlay.playturn(userInput)){
success
} else {
failure
}

Reply:
You could also switch out the while loop for a do while, that checks the condition at the end, that way you would not need to check isHasOne again internally.